platform/linuxbsd/export/export.cpp

```diff
@@ -31,27 +31,27 @@
 #include "export.h"
 
 #include "editor/editor_export.h"
-#include "platform/x11/logo.gen.h"
+#include "platform/linuxbsd/logo.gen.h"
 #include "scene/resources/texture.h"
 
-void register_x11_exporter() {
+void register_linuxbsd_exporter() {
 
 	Ref<EditorExportPlatformPC> platform;
 	platform.instance();
 
-	Ref<Image> img = memnew(Image(_x11_logo));
+	Ref<Image> img = memnew(Image(_linuxbsd_logo));
 	Ref<ImageTexture> logo;
 	logo.instance();
 	logo->create_from_image(img);
 	platform->set_logo(logo);
-	platform->set_name("Linux/X11");
+	platform->set_name("LinuxBSD");
```

Comment:
This should not be changed in this PR. The user-facing name should probably remain Linux for now.

Reply:
If it is changed, this could probably be "Linux/BSD" or "Linux&BSD", since I don't think special characters are a problem here.